sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientBuilder.java

```diff
@@ -259,6 +298,7 @@ public KeyEncryptionKeyClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link KeyEncryptionKeyClientBuilder} object.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the `Configuration` object. Should we add a note like you did in other methods on this same class?